### PR TITLE
Additional step in create frontend-app - Closes 1590

### LINKED ISSUE #1590: Add an additional step in `Create a user interface`
### Problem
In the [Create a user interface guide](https://lisk.com/documentation/integrate-blockchain/create-user-interface.html) we tell the reader to install dependencies at [2. Install Dependencies](https://lisk.com/documentation/integrate-blockchain/create-user-interface.html#install-dependencies). 

After we ask the user to add libraries in the `package.json` for `BigInt` compatibility, we don't tell the reader to run the `npm i` command again which is necessary. There is a case, where users don't run the said command and as a result, the project doesn't compile successfully.

### Solution
Add a step right after updating the `package.json` file and tell the reader to run the `npm i` command to successfully install the relevant dependencies.

Affected branch: `main/development`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1590

### CODE REVIEW

File: docs/modules/ROOT/pages/integrate-blockchain/create-user-interface.adoc [hunk 1]
```diff
@@ -129,7 +129,12 @@ To use `BigInt` in the frontend, it may be required to add the following options
   // [...]
 }
 ----
+After updating the `package.json` file, install the dependencies in the `frontend-app`:
 
+[source,js]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/integrate-blockchain/create-user-interface.adoc]:
```suggestion
[source,bash]
```

**Review by Tschakki — CHANGES_REQUESTED** [review]
Just a small correction for using the right code highlighting :)

**Review by chris529 — APPROVED** [review]
LGTM 👍

**Review by Tschakki — APPROVED** [review]
Thanks!

**Pull request merged**